docs/InstallEbpf.md

```diff
@@ -54,9 +54,9 @@ to install eBPF on all Windows nodes in a Kubernetes cluster.
 
 1. Deploy the binaries to `C:\Temp` on the machine (Windows Host) where you built the binaries.
    Start an admin Powershell on the Windows Host and do `.\x64\debug\deploy-ebpf`.
-   
-2. Build ebpf-for-windows image. 
-     
+
+2. Build ebpf-for-windows image.
```

Comment:
nit: markup allows numbered list items to be all "1.", which will be automatically enumerated upon rendering (makes it easier to maintain when needing to add/remove items without renumbering - exception is when cross-referencing between bullets is needed).

Reply:
Oops, missed your comments. Will fix it in a subsequent PR.